src/lib-components/CardMeta.vue

```diff
@@ -1,20 +1,27 @@
-<script setup lang="ts">
+<script
+  setup
+  lang="ts"
+>
 import { computed } from 'vue'
 import { useRoute } from 'vue-router'
 import type { PropType } from 'vue'
+import { useTheme } from '@/composables/useTheme'
 
 // COMPONENTS
 import SmartLink from '@/lib-components/SmartLink.vue'
 import IconWithLink from '@/lib-components/IconWithLink.vue'
 import RichText from '@/lib-components/RichText.vue'
+import BlockTag from '@/lib-components/BlockTag.vue'
+
+// import BlockTag from '@/lib-components/BlockTag.vue'
 
 // UTILITY FUNCTIONS
 import formatTimes from '@/utils/formatEventTimes'
 import formatDates from '@/utils/formatEventDates'
 
 // import removeHtmlTruncate from '@/utils/removeHtmlTruncate'
 
-import type { LocationItemType } from '@/types/types'
+import type { LocationItemType, ScreeningTagItemType } from '@/types/types'
```

Comment:
what is a ScreeningTagItemType? What naming pattern are we following? Should it be more like what it is in craft, like the craft field itself?

Reply:
The craft field is screeningTag

Reply:
![Screenshot 2024-07-01 at 10 47 13 PM](https://github.com/UCLALibrary/ucla-library-website-components/assets/751697/6157fd86-a20e-4732-9e73-1d04bcd87c00)